frontend-cert/js-projects/first-oop-lab/user-stories.md

```diff
@@ -0,0 +1,25 @@
+You should define a constant of type ENUM named `projectStatus` with the different project statuses you want to have such as `PENDING`, `SUCCESS`, or `FAILURE` with ypur preferred descriptions.
+
+You should define a class named `ProjectIdea` with a `constructor` that takes a `title` and a `description`, both strings. The class should also have a property named `status` also of type string.
+
+You should define a method named `updateProjectStatus` inside the `ProjectIdea` class that takes accepts a string `newStatus` as parameter and can update the project idea status.
+
+You should define a `ProjectIdeaBoard` class with a `constructor` that accepts a string `title` and initializes an empty array named `ideas` to hold instances of the `ProjectIdea` class.
+
+You should define a method named `pin` inside the `ProjectIdeaBoard` class that accepts an instance of the `ProjectIdea` class and pushes the given instance to the `ideas` array.
+
+You should define a method named `unpin` inside the `ProjectIdeaBoard` class that accepts an instance of the `ProjectIdea` class and removes the given instance from the `ideas` array.
+
+You should define a method named `count` that returns the number of project ideas in a given project idea board object.
+
+You should define a method named `toString` that returns the name of the projects ideas, their description and status as a nicely formatted string.
+
+You should create one or more instance of the `ProjectIdeaBoard` class.
+
+You should create one or more instance of the `ProjectIdea` class and `pin` each of them to any one of the `ProjectIdeaBoard` instances.
+
+You should not `pin` one instance ot the `ProjectIdea` class to multiple instances of the `ProjectIdeaBoard` class.
+
+You should `unpin` random instances of the `ProjecIdea` class from their respective boards.
+
+You should examine the boards before and after pinning and unpinning of new project ideas.
```

Comment:
I am not sure what this last user story means

Reply:
I have removed these two. I think they are not that important.